### PR TITLE
Enable artifact registry and a couple tweaks

### PR DESCRIPTION
**Pull request by andrewplummer**

### CODE REVIEW

File: src/cloud/bootstrap.js [hunk 1]
```diff
@@ -53,6 +53,17 @@ export async function bootstrapProjectEnvironment(project, environment, config)
   await execSyncInherit('gcloud services enable compute.googleapis.com');
   console.info(yellow('=> Enabling Kubernetes services'));
   await execSyncInherit('gcloud services enable container.googleapis.com');
+  await execSyncInherit('gcloud services enable artifactregistry.googleapis.com');
+  await execSyncInherit('gcloud services enable cloudbuild.googleapis.com');
+
+  try {
+    // TODO: better way than hardcoding the location?
+    await execSyncInherit(
+      'gcloud artifacts repositories create default --repository-format=docker --location=us-east1'
```
Comment (beatlevic) [inline src/cloud/bootstrap.js:62]: The location depends on the the project setup vars

File: src/cloud/build.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ async function buildImageRemote(options) {
 
   const flags = [
     `--config ${path.resolve(__dirname, 'cloudbuild.yaml')}`,
-    `--substitutions _IMAGE=${image},_DOCKERFILE=${dockerfile}`,
+    `--substitutions _IMAGE=${image},_DOCKERFILE=${dockerfile},_LOCATION=us-east1,_REPOSITORY=default`,
```
Comment (beatlevic) [inline src/cloud/build.js:31]: Location is different for each project

File: src/cloud/index.js [hunk 3]
```diff
@@ -64,7 +64,7 @@ export async function account(options) {
 export async function login() {
   console.info(
     yellow(
-      'This will open a glcoud login URL in your browser twice. First for your account auth, and a second time for your application default.'
+      'This will open a gcloud login URL in your browser twice. First for your account auth, and a second time for your application default.'
```
Comment (beatlevic) [inline src/cloud/index.js:67]: You don't seem to like `glcoud`, but I agree that `gcloud` is actually correct.

File: src/cloud/push.js [hunk 4]
```diff
@@ -3,11 +3,12 @@ import kleur from 'kleur';
 import { exec, execSyncInherit } from '../util/shell';
 
 async function pushImage(project, image, tag) {
-  const gcrTag = `gcr.io/${project}/${image}:${tag}`;
-  console.info(kleur.green(`Pushing ${gcrTag}`));
-  await exec(`docker tag ${image}:${tag} ${gcrTag}`);
+  // TODO: don't hardcode location
```
Comment (beatlevic) [inline src/cloud/push.js:6]: Ah, I see the location TODO! You can fetch it from the config.json

**Comment by andrewplummer** [comment]
ok I'm closing this for now after looking at it I don't see any added benefit to artifact registry plus it costs slightly more... we can revisit this if we have reason to

**Pull request closed without merging**